src/main/xar-resources/data/upgrading.xml

```diff
@@ -3,7 +3,7 @@
 <book>
     <bookinfo>
         <productname>eXist-db – Open Source Native XML Database</productname>
-        <title>Upgrading Guide</title>
+        <title>Upgrade Guide</title>
         <date>November 2009</date>
```

Comment:
Should we update the date here, I feel that last changed date is more important here